nextjs/src/app/data-sources/ExternalDataSourceUpdates.tsx

```diff
@@ -0,0 +1,145 @@
+"use client";
+
+import { gql, useQuery } from "@apollo/client";
+import { ActionNetworkLogo, AirtableLogo } from "@/components/logos";
+import { Button } from "@/components/ui/button";
+import { Skeleton } from "@/components/ui/skeleton";
+import Link from "next/link";
+import { AutoUpdateCard } from "@/components/AutoUpdateCard";
+import { DataSourceType, ListExternalDataSourcesQuery, ListExternalDataSourcesQueryVariables } from "@/__generated__/graphql";
+import { useEffect } from "react";
+import qs from 'query-string'
+
+const LIST_UPDATE_CONFIGS = gql`
+  query ListExternalDataSources {
+    externalDataSources {
+      id
+      name
+      dataType
+      connectionDetails {
+        crmType: __typename
+      }
+      autoUpdateEnabled
+      jobs {
+        lastEventAt
+        status
+      }
+      updateMapping {
+        source
+        sourcePath
+        destinationColumn
+      }
+    }
+  }
+`;
+
+export default function ExternalDataSourceUpdates() {
+  const { loading, error, data, refetch } = useQuery<ListExternalDataSourcesQuery, ListExternalDataSourcesQueryVariables>(LIST_UPDATE_CONFIGS);
+
+  useEffect(() => {
+    refetch()
+  }, [refetch])
+
+  return (
+    <div className=" max-w-7xl space-y-7 w-full">
+      <PageHeader />
+      <div className="border-b border-meepGray-700 pt-10" />
+      <header className='flex flex-row justify-end'>
+        <h2 className="text-hSm mr-auto">Your team’s membership lists</h2>
```

Comment:
Is it the case that they are always and everywhere membership lists?

Reply:
Yes, due to the filter just below https://github.com/commonknowledge/meep-intelligence-hub/pull/24/files/ef5d38066294506d56d5ee912df5c45f3448e26f#diff-ffddb610c0a8b276a27e83d512c8df1fbfc1aa414de32c0ed81f183802b73cfdR63